sdcm/stress_thread.py

```diff
@@ -122,7 +123,9 @@ def create_stress_cmd(self, node, keyspace_idx):
             stress_cmd += \
                 ' -transport "truststore=/etc/scylla/ssl_conf/client/cacerts.jks truststore-password=cassandra"'
 
-        if self.node_list and '-node' not in stress_cmd:
+        if (connection_bundle_file := self.node_list[0].parent_cluster.connection_bundle_file) and '-node' not in stress_cmd:
+            stress_cmd += f" -cloudconf file={Path('/tmp') / connection_bundle_file.name}"
```

Comment:
Is it really correct -> `-cloudconf file=...` ?

Reply:
yeah, it's c-s stupid command argument setup... took me a while to figure that out.